drivers/net/Kconfig

```diff
@@ -88,7 +88,7 @@ config TELNET_IOTHREAD_PRIORITY
 
 config TELNET_IOTHREAD_STACKSIZE
 	int "I/O thread stack size"
-	default 1024
```

Comment:
It is inappropriate to increase the size of this tuned stack size.  Let's not do that.